studio/components/layouts/TableEditorLayout/TableEditorMenu.tsx

```diff
@@ -74,11 +74,13 @@ const TableEditorMenu: FC<Props> = ({
     setIsRefreshing(false)
   }
 
-  const schemas = meta.schemas.list()
-  const tables = meta.tables.list((table: PostgresTable) => table.schema === selectedSchema)
+  const schemas: any[] = meta.schemas.list()
```

Comment:
perhaps these could be strongly typed - 

https://github.com/supabase/postgres-meta/blob/511b735757fe3527347db818e7e060809bfcb892/src/lib/types.ts#L218

Reply:
Updated PR